docs/building-deployer-helm.md

```diff
@@ -610,22 +611,22 @@ general
 [onboarding guide](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment#product-identifiers).
 See [this section](building-deployer.md#publishing-a-new-version)
 for more information about tags. Let's say your application has a
-track named `v6.0.0`.
+track named `9.0`.
```

Comment:
maybe add:

```suggestion
track named `9.0`, and that the current release for this track is `9.0.3`.
```

Reply:
@gibbleyg Just to bring it to your attention, there is this comment as well. It's very easy to miss the comments that get sent the same time as the approval.